NDTensors/test/ITensors/TestITensorDMRG/dmrg.jl

```diff
@@ -0,0 +1,28 @@
+using Test
+using ITensors
+using NDTensors
+using Random
+
+Random.seed!(1234)
```

Comment:
I would move this to be right above the line:
```julia
psi0 = dev(randomMPS(elt, sites;))
```
so it is consistent every time you run `test_dmrg`.